extensions/frontend-tiles/package.json

```diff
@@ -11,11 +11,12 @@
     "build:cjs": "tsc 1>&2 --outDir lib/cjs",
     "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
     "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "cover": "npm -s test",
     "docs": "",
     "extract-api": "betools extract-api --entry=frontend-tiles",
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
-    "test": "",
-    "cover": ""
+    "test": "npm run -s webpackTests && certa -r chrome",
```

Comment:
can you add `.npmignore` or `files` section in the pkg.json so we dont deliever the tests in the published pkg?

Reply:
added .npmignore.